protob/messages.proto

```diff
@@ -912,6 +922,102 @@ message CosiSignature {
 	optional bytes signature = 1;		// Signature
 }
 
+///////////////////
+// Iota messages //
+///////////////////
+
+/**
+ * Request: Show seed on the device
+ * @next Success
+ * @next Failure
+ */
+message IotaShowSeed {
```

Comment:
TREZOR never shows the seed after it is backed up by the user.

Reply:
This is not a trezor seed, but the IOTA seed. As long as I don't have a computer program that converts 24 words into the IOTA seed the same way, I'd like to display the seed on the screen.

Reply:
Then I think it is a good idea to write a program to convert the mnemonic into the IOTA seed because the TREZOR never gives out private keys after initial backup

Reply:
It's on the todo list. If it blocks merging, I'll remove the option.

Reply:
I guess it does not make sense to merge in the message which will be removed afterward. So let's remove the message.

Reply:
Ok